packages/nimble-components/src/table/specs/table-column-specs/table-column-mapping.md

```diff
@@ -102,30 +103,14 @@ _Props/Attrs_
 
 -   `field-name`: string
 -   `key-type`: 'string' | 'number' | 'boolean'
+-   `label-hidden`: boolean - When `true`, the `label` for a mapping will be rendered in a cell next to the specified icon. Note, that this attribute only affects what is rendered in a cell; a group row will always render both the icon and the label.
+-   `icon-width`: boolean - When `true`, the column will have a fixed width that makes the column the appropriate width to render only a single icon in the cell. This should only be set to `true` when the header contains a single icon (no text) and `label-hidden` is `true`. If set to `true` when `label-hidden` is `false`, the column's configuration will be considered invalid.
```

Comment:
I'm open to suggestions about the name of this attribute. Particularly, I want to convey that when this attribute is set the column will not be resizable and will be very narrow. I also what to avoid confusion with anything more generic we add to columns to make them fixed-width/not resizable/pixel width.
	
Some other ideas I thought about were:
- fixed-width: I decided against this name because I didn't think it properly conveyed that the column would be narrow. I also thought it could be confused with more generic column sizing features we might add in the future.
- icon-only: This sounds more about what is being displayed than the size.
- fixed-icon-width: More verbose than `icon-width` with no clear advantage. It also sounds more like the icon is fixed width than the column.

Reply:
Combining conversation threads, @m-akinc posted [here](https://github.com/ni/nimble/pull/1992#discussion_r1546957248): 
>I have reservations about the name icon-width, because without proper context, it strongly implies a numeric value controlling the width of an icon (or icons). I realize that having the word "icon" in there is important, because naming it something like fixed-width would sound like it could be used even when the column/header contained text. Maybe icon-fixed-width?

Reply:
I'm not opposed to `icon-fixed-width` either, but for now, I've updated my proposal to `fixed-to-icon-width`. It's a bit verbose, but I think it makes it more explicit that nothing about the icon is "fixed width" -- instead it's the column that is fixed to an icon's width. I also thought about using the term "locked" rather than "fixed" (i.e. `locked-to-icon-width`).

I'm still happy to take more opinions on this name.